graph_weather/models/atmorep/training/loss.py

```diff
@@ -0,0 +1,145 @@
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class MaskedReconstructionLoss(nn.Module):
+    def __init__(self, config):
```

Comment:
This should take all the arguments it needs in the `init`, not the overall `config`.